integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/DorisContainer.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.container;
+
+import static java.lang.String.format;
+
+import com.google.common.collect.ImmutableSet;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.rnorth.ducttape.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.Network;
+
+public class DorisContainer extends BaseContainer {
+  public static final Logger LOG = LoggerFactory.getLogger(DorisContainer.class);
+
+  public static final String DEFAULT_IMAGE = System.getenv("GRAVITINO_CI_DORIS_DOCKER_IMAGE");
+  public static final String HOST_NAME = "gravitino-ci-doris";
+  public static final String USER_NAME = "root";
+  public static final String PASSWORD = "root";
+  public static final int FE_HTTP_PORT = 8030;
+  public static final int FE_MYSQL_PORT = 9030;
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  protected DorisContainer(
+      String image,
+      String hostName,
+      Set<Integer> ports,
+      Map<String, String> extraHosts,
+      Map<String, String> filesToMount,
+      Map<String, String> envVars,
+      Optional<Network> network) {
+    super(image, hostName, ports, extraHosts, filesToMount, envVars, network);
+  }
+
+  @Override
+  protected void setupContainer() {
+    super.setupContainer();
+    withLogConsumer(new PrintingContainerLog(format("%-14s| ", "DorisContainer")));
+    withStartupTimeout(Duration.ofMinutes(10));
```

Comment:
why set the timeout as 10 minutes? Seems a bit long

Reply:
seems 10 minutes is not enough. Doris Docker image size is too big, docker pull for github CI will cost too many time.

Maybe I'd better use old Doris version to reduce docker image size

Reply:
Based on this scenario, I believe we may need to cache the image in CI. see https://github.com/datastrato/gravitino/issues/998 or https://github.com/datastrato/gravitino/issues/1142

Reply:
According to the log, the root cause is that the service can't be launched within 10 minutes, please check it again
[catalog-jdbc-doris-integration-test.log](https://github.com/datastrato/gravitino/files/14681505/catalog-jdbc-doris-integration-test.log)

Reply:
@yuqi1129 thanks, I will fix it later.